js/src/personal/tony/Tony.page.tsx

```diff
@@ -1,35 +1,41 @@
-import { Title, Text } from '@mantine/core'
+// TonyPage component
```

Comment:
It's good to have comments describing each component. It's better to put this right above the `export function TonyPage()` instead of above the imports. Some IDEs auto-collapse imports and may miss the comment.

Also for that comment for the component, I find JSDoc format better.
`/** Page component for the main page of Tony's Website */`